src/icons/item-icon/ItemIcon.tsx

```diff
@@ -81,7 +81,16 @@ export interface ItemIconProps {
 
 const ItemIcon = ({ className, dimension = 32, iconType, title }: ItemIconProps) => {
     const IconComponent = (itemIconTable as { [key: string]: React.FC<SVGProps> })[iconType] || IconFileDefault;
-    return <IconComponent className={className} height={dimension} title={title} width={dimension} />;
+    return (
+        <IconComponent
+            className={className}
+            height={dimension}
+            title={title}
+            width={dimension}
+            aria-hidden="true"
+            role="presentation"
```

Comment:
It looks like this should already be handled by the AccessibleSVG component if a title is not passed in: https://github.com/box/box-ui-elements/blob/master/src/icons/accessible-svg/AccessibleSVG.tsx#L29